internal/podman/client.go

```diff
@@ -118,13 +149,22 @@ func (o *client) Diagnostics(ctx context.Context) *Diagnostic {
 		return d
 	}
 
+	d.Version, err = o.Version(ctx)
+	if err != nil {
+		d.VersionFound = false
```

Comment:
1. do we need VersionFound? (which is equivalent to  d.Version = nil)
2. only a question, what do you think about collecting the errs and continuing the collect diagnostics flow? e.g.
if err != nil {
     d.Errors = append(d.Errors, fmt.Errorf("failed to collect podman version: %w", err))
}
same for the errs below